torch/csrc/jit/autodiff.cpp

```diff
@@ -28,6 +32,33 @@ bool isDifferentiable(Graph & g) {
 }
 
 
+static std::vector<SymbolicVariable> mm_grads(std::vector<SymbolicVariable> inputs,
+                                             std::vector<SymbolicVariable> grads) {
+  SymbolicVariable dmat1, dmat2;
+  if (auto type = inputs.at(0).value()->type()->cast<TensorType>()) {
+    auto sizes = type->sizes(), strides = type->strides();
+    if (strides.at(0) == 1 && strides.at(1) == sizes.at(0)) {
+      dmat1 = inputs.at(1).mm(grads.at(0).t()).t();
+    } else {
+      dmat1 = grads.at(0).mm(inputs.at(1).t());
+    }
+  } else {
+    dmat1 = grads.at(0).mm(inputs.at(1).t());
+  }
+  if (auto type = inputs.at(1).value()->type()->cast<TensorType>()) {
+    auto sizes = type->sizes(), strides = type->strides();
+    if (strides.at(0) == 1 && strides.at(1) == sizes.at(0)) {
+      dmat2 = grads.at(0).t().mm(inputs.at(0)).t();
+    } else {
+      dmat2 = inputs.at(0).t().mm(grads.at(0));
+    }
+  } else {
+    dmat2 = grads.at(0).mm(inputs.at(1).t());
```

Comment:
There's a typo here (should be `dmat2 = inputs.at(0).t().mm(grads.at(0));`), the same typo is in the current version in autodiff.cpp.